utils/restUtils.js

```diff
@@ -1,8 +1,21 @@
-import { AuthenticationError, UserInputError, ServerError } from 'apollo-server-express';
+import { AuthenticationError, UserInputError, ApolloError } from 'apollo-server-express';
+import logger from './logger';
 /*
 convert the REST status codes to GQL errors, or return the response if passing
 */
 export const restErrorResponseHandler = async response => {
+  // Generic handle 5xx errors
+  if (response.status >= 500 && response.status <= 599) {
+    let responseBody;
+    try {
+      responseBody = await response.json();
+    } catch {
+      responseBody = { message: '' };
+    }
+    logger.debug(`Server 5xx response: ${JSON.stringify(responseBody)}`);
+    throw new ApolloError(responseBody.message || '', response.status);
```

Comment:
I don't know if passing the downstream error message through to the caller is a good idea

Reply:
fair enough, how about the http error code

Reply:
depends what the UI is expecting honestly and what the graphql standard errors for such cases. I'm not sure if the response status code is going to be interpreted by them .. Jon and Minh can provide better opinion

Reply:
I've removed all error info from going to downstream... we can add later if needed.